.travis.yml

```diff
@@ -137,6 +137,12 @@ matrix:
         - make arminstall
         - make armfuzz
 
+    - name: ARM Build Test (on Trusty)
+      dist: trusty
+      script:
+        - make arminstall
+        - make armbuild
```

Comment:
The specific issue that motivates this change is a build issue with an old toolchain, not a runtime problem. So in the interests of saving time, I've only built zstd, not run any tests. Does that seem reasonable? Should we do `armtest` instead?

Reply:
`armbuild` is fine, it answers the issue.

Maybe a comment reminding why this test is here would be useful.
Without context, it's easy to wonder why this test is not simply folded into the one just above.